src/ui/server/controllers/insurance/declarations/anti-bribery/code-of-conduct/index.ts

```diff
@@ -61,6 +62,9 @@ export const get = (req: Request, res: Response) => {
     ...pageVariables(refNumber),
     userName: getUserNameFromSession(req.session.user),
     application: mapApplicationToFormFields(res.locals.application),
+    applicationAnswer: application.declaration[FIELD_ID],
+    horizontalRadios: true,
+    HINT_HTML: 'partials/code-of-conduct-hint.njk',
```

Comment:
Constants, also could these additions be in `pageVariables` to be DRY?  At least for `horizontalRadios` `X_HTML`